.github/workflows/lint.yml

```
@@ -0,0 +1,47 @@
+
+      - name: 의존성 설치
+        run: npm ci
+
+      - name: ESLint 실행
+        run: npm run lint
+        # -- 만약 lint 오류가 있으면 이 단계에서 실패 → PR에 에러 표시됨
+
+  lint-frontend:
+    name: Frontend Lint
+    runs-on: ubuntu-latest
+    working-directory: ./frontend         # frontend 서브폴더 지정
```

actionlint (1.7.7)

unexpected key "working-directory" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)